Clinical-Genomics/scout
# Fixed javascript code and moved css code in case page

Fix #2280 

**How to test**:
1. From master branch, make sure that javascript code doesn't work (see bug issue description).
2. Install this branch and confirm that the errors are fixed

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
